### ISSUE TITLE
Open the API

### ISSUE THREAD

**Issue by ArneBachmann** [erikbern/git-of-theseus#51]
Allow manual provision of arguments to `analyze.analyze()` instead of getting them only from `sys.argv`.
Now I have to fiddle with `sys.argv` at runtime to invoke `analyze()`.

**Comment by erikbern** [comment]
Yeah seems like a great idea. Want to put together a pull request?

**Comment by ArneBachmann** [comment]
Will try to, time allowing.
